recipes/javafx/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "javafx-sdk" %}
+{% set version = "11.0.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/javafxports/openjdk-jfx/archive/11.0.4+1.tar.gz
+  sha256: f03d16adae701406ff9e1fc05e461a327ca4f97b5bf18b0d727037bb425ec25a
+
+build:
+  number: 0
+  skip: true  # [win]
+  missing_dso_whitelist:
+    - '*'
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - bison
+    - flex
+    - gperf
+    - pkgconfig
+    - gtk2
+    - pango
+    - freetype
+    - ant
+    - libx11-devel-cos6-x86_64
+    - libxxf86vm-devel-cos6-x86_64
+    - mesa-libgl-devel-cos6-x86_64
+    - libpng-devel-cos6-x86_64
+    - libpng-devel-cos6-x86_64
+  host:
+    - coreutils >=8
+    - openjdk >=11
+
+test: 
+  commands:
+    - test -f "${PREFIX}/lib/${PKG_NAME}-${PKG_VERSION%}/lib/javafx.base.jar"
+
+about:
+  home: https://gluonhq.com/products/javafx/
+  license: GPL-2.0 + Classpath
+  license_file: LICENSE.txt
```

You can remove the licence file from the recipe directory and get it from the source tarball instead:
```suggestion
  license_file: LICENSE
```